coffee_cmd/src/cmd.rs

```diff
@@ -53,6 +53,9 @@ pub enum CoffeeCommand {
     /// show the README file of the plugin
     #[clap(arg_required_else_help = true)]
     Show { plugin: String },
+    /// clean up remote repositories storage information
+    #[clap(arg_required_else_help = false)]
```

Comment:
```suggestion
    #[clap(arg_required_else_help = true)]
```

Reply:
Should coffee nurse command take any arguments?

Reply:
Ah now I got it, no your are right!